Countly.d.ts

```diff
@@ -0,0 +1,824 @@
+declare module 'countly-sdk-react-native-bridge' {
+
+  interface CountlyEventOptions {
+    eventName?: string;
+    eventCount?: number;
+    eventSum?: number;
+    segments?: Record<string, any>;
+  }
+
+  interface CountlyUserDataBulk {
+    save:() => Promise<void>;
+    setProperty:(keyName: string, keyValue: any) => Promise<string> | string;
+    increment:(keyName: string) => Promise<string> | string;
+    incrementBy:(keyName: string, keyValue: any) => Promise<string> | string;
+    multiply:(keyName: string, keyValue: any) => Promise<string> | string;
+    saveMax:(keyName: string, keyValue: any) => Promise<string> | string;
+    saveMin:(keyName: string, keyValue: any) => Promise<string> | string;
+    setOnce:(keyName: string, keyValue: any) => string | void;
+    pushUniqueValue:(keyName: string, keyValue: any) => string | void;
+    pushValue:(keyName: string, keyValue: any) => string | void;
+    pullValue:(keyName: string, keyValue: any) => string | void;
+  }
+
+  type CountlyCallback = (message: string) => void;
+
+  interface FeedbackWidget {
+    id: string;
+    type: string;
+    name?: string;
+  }
+
+  type WidgetCallback = () => void;
+
+  type CustomMetric = {
+    [key: string]: string;
+  };
+
+  type TraceCustomMetric = {
+    [key: string]: string | number;
+  };
+
+  interface NetworkTraceData {
+    networkTraceKey: string;
+    responseCode: number;
+    requestPayloadSize: number;
+    responsePayloadSize: number;
+    startTime: number;
+    endTime: number;
+  }
+
+  type ValidationFunction = (
+    stringValue: string,
+    stringName: string,
+    functionName: string
+  ) => Promise<string | null>;
+
+  type ResultObject = (
```

Comment:
I checked this at the example we have and it was giving an error. Changing it to this fixed it:
  type ResultObject = {
    error: string,
    data: object,
  };
So can you go through the example app and check the errors coming through and fix them.